src/Stratis.Bitcoin.Features.WatchOnlyWallet/WatchOnlyWalletManager.cs

```diff
@@ -64,9 +72,22 @@ public void Initialize()
             // load the watch only wallet into memory
             this.Wallet = this.LoadWatchOnlyWallet();
 
+            this.signals.OnBlockConnected.Attach(this.onBlockConnected);
+            this.signals.OnTransactionReceived.Attach(this.onTransactionAvailable);
+
             this.LoadTransactionLookup();
         }
 
+        private void onTransactionAvailable(Transaction transaction)
+        {
+            this.ProcessTransaction(transaction);
+        }
+
+        private void onBlockConnected(ChainedHeaderBlock chainedheaderblock)
```

Comment:
@noescape00 can you Pascal case this everywhere? I.e. it should be `private void OnBlockConnected`

Reply:
Fixed